ghmirror/data_structures/redis_data_structures.py

```diff
@@ -50,7 +51,9 @@ def __getitem__(self, item):
     def __setitem__(self, key, value):
         sr_key = self._serialize(key)
         sr_value = self._serialize(value)
-        self.wr_cache.set(sr_key, sr_value)
+        rand_val = randint(1, 4320)
+        # set entry to expire in rand_val hrs
```

Comment:
it would be more useful for the comment to explain the 4320 above instead of just reading the code below

Reply:
ha you're right. i had that in before and accidentally deleted it